pkgs/development/tools/rust/rustup/default.nix

```diff
@@ -39,7 +39,9 @@ rustPlatform.buildRustPackage rec {
     )
   ];
 
-  doCheck = !stdenv.isAarch64 && !stdenv.isDarwin;
+  # Disable tests until they can be run with --features no-self-update
```

Comment:
```
checkType = "debug";
```

Reply:
The tests fail when `no-self-update` is enabled, so I don’t think this will fix it?
I’ll create an issue upstream in the next days.

Reply:
right.